client.go

```diff
@@ -170,3 +171,92 @@ func Exec(ctx context.Context, cmd string, args ...interface{}) error {
 func Query(ctx context.Context, cmd string, args ...interface{}) Args {
 	return DefaultClient.Query(ctx, cmd, args...)
 }
+
+type MultiError []error
```

Comment:
It should work, but then we lose the additional errors. However, given that we don't use it anywhere (except to stringify the error), I think that's a fair compromise. We can always expose it later. Will change it.